### ISSUE TITLE
Where can we download evaluation dataset files

### ISSUE THREAD

**Issue by yujunwei04** [Zhao-Yian/GraCo#8]
Hi GraCo authors,

Thank you for the impressive work! Could you please share the evaluation datasets raw files in https://github.com/Zhao-Yian/GraCo/blob/main/config.yml for benchmarking purpose?

```
# Evaluation datasets
GRABCUT_PATH: "/path/to/datasets/GrabCut"
BERKELEY_PATH: "/path/to/datasets/Berkeley"
DAVIS_PATH: "/path/to/datasets/DAVIS"
PARTIMAGENET_PATH: "/path/to/datasets/PartImageNet"
SA1B_PATH: "/path/to/datasets/SAM"
PASCALPART_PATH: "/path/to/datasets/VOCdevkit/VOC2010"
BraTS_PATH: "/path/to/datasets/BraTS20"
ssTEM_PATH: "/path/to/datasets/ssTEM/stack1"
OAIZIB_PATH: "/path/to/datasets/OAI-ZIB"
PASCALVOC_PATH: "/path/to/datasets/VOCdevkit/VOC2010"
```

**Comment by Zhao-Yian** [comment]
you can refer to the repo of SimpleClick to get these datasets' links: https://github.com/uncbiag/SimpleClick

**Comment by yujunwei04** [comment]
Thanks for your reply! 

May I ask for the file ```self.dataset_path / f'ImageSets/Main/part_{self.dataset_split}.txt'``` in line 174 of [https://github.com/Zhao-Yian/GraCo/blob/main/isegm/data/datasets/pascalpart.py](url)

AND the file ```os.path.join(self.dataset_path, "class_names.json")``` in line 25 of [https://github.com/Zhao-Yian/GraCo/blob/main/isegm/data/datasets/partimagenet.py](url)

It seems these files are not from the official pascalpart and partimagenet dataset.

Thank you!!

**Comment by Zhao-Yian** [comment]
The purpose of using `ImageSets/Main/part_{self.dataset_split}.txt` is to filter out the images of the SBD validation set included in the training set, but this has little effect. You can directly use the official file, i.e., modify it to `ImageSets/Main/{self.dataset_split}.txt`.
The file `class_names.json` is a mapping of the ids and class labels of the partimagenet dataset, and I will update it to the repo later.

**Comment by yujunwei04** [comment]
> The purpose of using `ImageSets/Main/part_{self.dataset_split}.txt` is to filter out the images of the SBD validation set included in the training set, but this has little effect. You can directly use the official file, i.e., modify it to `ImageSets/Main/{self.dataset_split}.txt`. The file `class_names.json` is a mapping of the ids and class labels of the partimagenet dataset, and I will update it to the repo later.

Hi Yian, may I ask the status of uploading the `class_names.json` of the PartImageNet dataset? Thank you!
